### ISSUE TITLE
"Could not authorise application" error box does not have "OK" button

### ISSUE THREAD

**Issue by tbm** [laurent22/joplin#2225]
When I click on "Synchronise" without supplying valid Dropbox information, I get a "Could not authorise application" error box.  This is fine, but the problem is that the error box has no "OK" or other button to make it go away.  (Eventually I figured out that pressing Escape works, but this is not obvious).

## Environment

Joplin version: 1.0.176
Platform: Linux
OS specifcs: Debian, but using AppImage from GitHub

## Steps To Reproduce

```
rm -rf ~/.config/joplin-desktop
./Joplin-1.0.176-x86_64.AppImage
```

Click on Synchronise

Step 2: Enter the code provided by Dropbox: leave that field empty

You'll get an error window:

```
Could not authorise application:
(json info)
Please try again.
```

There's no button that makes the error window go away.

Describe what you expected to happen:

There should be a button so you can close the window.

**Comment by tbm** [comment]
There's no reason to believe it has been fixed.

**Comment by mic704b** [comment]
It might have been fixed by #2410 ?

**Comment by tbm** [comment]
@mic704b thanks, this is indeed working now! Thanks
